test/reverse.js

```diff
@@ -43,7 +43,7 @@ describe('Reverse feature', function () {
 
       applyPatch (element, fragment) {
         childNodes = fragment.map(childNode => {
-          if (!childNode.style.top) {
+          if (!childNode.style.top || childNode.nodeName === 'tr') {
```

Comment:
What was this change about?

Reply:
Previously, the sizer element (`<tr>`) didn't have a `top` css property, which met the test's condition for exclusion. Now that it does have that property, the condition needed to be updated.

Reply:
`<tr>` can be changed via `scrollerTagName` so we should probably use that instead of hardcoding `tr`.